backend/api/community_create.go

```diff
@@ -0,0 +1,113 @@
+package api
+
+import (
+	"database/sql"
+	"errors"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	db "github.com/git-adithyanair/cs130-group-project/db/sqlc"
+	"github.com/git-adithyanair/cs130-group-project/token"
+)
+
+type CreateCommunityRequestStore struct {
+	Name    string  `json:"name" binding:"required"`
+	XCoord  float64 `json:"x_coord" binding:"required,numeric"`
+	YCoord  float64 `json:"y_coord" binding:"required,numeric"`
+	PlaceID string  `json:"place_id" binding:"required"`
+	Address string  `json:"address" binding:"required"`
+}
+
+type CreateCommunityRequest struct {
+	Name         string                        `json:"name" binding:"required"`
+	CenterXCoord float64                       `json:"center_x_coord" binding:"required,numeric"`
+	CenterYCoord float64                       `json:"center_y_coord" binding:"required,numeric"`
+	Range        int32                         `json:"range" binding:"required,numeric"`
```

Comment:
Should this be required? I was assuming we'd have a default value for this and if for some reason the admin wanted to change it they could.

Reply:
I think when they're creating a community they should provide one, and if not the frontend can select a default value